packages/compartment-mapper/test/policy.test.js

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
These can no longer be inferred from directory pattern matching? I do like that they’re evident, but also they are numerous.